### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by Dok11** [1988kramer/camera-pose#1]
Hi! Nice work!
I pretty detailed read your [article](http://andrewjkramer.net/camera-pose-estimation-using-convolutional-neural-networks/) and code at this repository. And I have some questions, I will happy if you can explain my trouble with understanding your code.

Why you dont use BatchNormailzation?
How you choose this convolution sizes? Why 11,11 but not 15,15 or any another?
https://github.com/1988kramer/camera-pose/blob/master/camera-pose-spp.py#L53

Now I try somthing similar neural network on my dataset (which I created in Blender 3d) and incresing conv size from 3,3 to 11,11 really increse quality of working my network.
![image](https://user-images.githubusercontent.com/2697890/63442826-7023ff80-c43c-11e9-8ee0-cc3c583c4c73.png)
its better than what I got past months while working on it

Why here this strange numbers? Where they from?
https://github.com/1988kramer/camera-pose/blob/master/camera-pose-spp.py#L74

Why you choose this decay? I never saw articles which explain this argument and still dont realize it =)
https://github.com/1988kramer/camera-pose/blob/master/camera-pose-spp.py#L108

PS Thank you for sharing you code and article
